rcl_logging_noop/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if(NOT WIN32)
 endif()
 
 set(${PROJECT_NAME}_sources
-  src/rcl_logging_noop/rcl_logging_noop.cpp
+  src/rcl_logging_noop.cpp
```

Comment:
this aligns with rcl_logging_spdlog directory tree.

Reply:
https://github.com/ros2/rcl_logging/pull/120/commits/7df68baf786810a7d57d4436bc5a37f60a296468 fixes.